Gravity-Notes/src/Components/TextArea.js/Textarea.js

```diff
@@ -68,16 +60,25 @@ export const Textarea = () => {
             required
           ></textarea>
         </p>
-        <button onClick={(e)=>addNoteHandler(e,note)}>
+        {note.flag 
+         ? 
+           <button >
+          <GrUpdate color="white" size={20} />
+          </button>
+         :
+          <button >
           <IoIosAdd size={35} />
-        </button>
+          </button>
+         }
+        
 
       </form>
 
     {/* Note card component */}
 
-          {console.log(noteState.notes)}
-          
+          {/* {console.log(noteState.notes)}
```

Comment:
Removed unused logs

Reply:
sure  & I will make these changes in UI soon, was going for functionality first and soon gonna refactor my UI

thanks, bro🙌